geograypher/cameras/cameras.py

```diff
@@ -748,47 +763,86 @@ def save_images(self, output_folder, copy=False, remove_folder: bool = True):
+
+        Returns:
+            List[Tuple[float, float] or Tuple[float, float, float]]: 
+                List of tuples containing the camera locations.
+        """
+        return [x.get_camera_location(**kwargs) for x in self.cameras]
 
     def get_subset_ROI(
         self,
-        ROI: Union[PATH_TYPE, gpd.GeoDataFrame],
-        buffer_radius_meters: float = 50,
-    ):
+        ROI: Union[PATH_TYPE, gpd.GeoDataFrame, Polygon, MultiPolygon],
+        buffer_radius: float = 0,
+        is_geospatial: bool = None
+    ):  
         """Return cameras that are within a radius of the provided geometry
 
         Args:
-            geodata (Union[PATH_TYPE, gpd.GeoDataFrame]): Geopandas dataframe or path to a geofile readable by geopandas
-            buffer_radius_meters (float, optional): Return points within this buffer of the geometry. Defaults to 50.
+            geodata (Union[PATH_TYPE, gpd.GeoDataFrame, Polygon, MultiPolygon]): 
+                This can be a Geopandas dataframe, path to a geofile readable by geopandas, or 
+                Shapely Polygon/MultiPolygon information that can be loaded into a geodataframe
+            buffer_radius (float, optional): 
+                Return points within this buffer of the geometry. Defaults to 0. Represents 
+                meters if ROI is geospatial.
+            is_geospatial (bool, optional): 
+                A flag for user to indicate if ROI is geospatial or not; if no flag is provided, 
+                the flag is set if the provided geodata has a CRS.
+        Returns:
+            subset_camera_set (List[PhotogrammetryCamera]): 
+                List of cameras that fall within the provided ROI
         """
-        if not isinstance(ROI, gpd.GeoDataFrame):
+        # construct GeoDataFrame if not provided
+        if isinstance(ROI, (Polygon,MultiPolygon)):
+            # assume geodata is lat/lon if is_geospatial is True
+            if is_geospatial:
+                ROI = gpd.GeoDataFrame(crs='epsg:4326', geometry=[ROI])
+            else:
+                ROI = gpd.GeoDataFrame(geometry=[ROI])
+        elif not isinstance(ROI, gpd.GeoDataFrame):
             # Read in the geofile
             ROI = gpd.read_file(ROI)
-
-        # Make sure it's a geometric (meters-based) CRS
-        ROI = ensure_projected_CRS(ROI)
+        
+        if is_geospatial is None:
+            is_geospatial = ROI.crs is not None
+
+        if not is_geospatial:
+            # get internal coordinate system camera locations
+            image_locations = [Point(*x) for x in self.get_camera_locations()]
+            image_locations_df = gpd.GeoDataFrame(geometry=image_locations)
+        else:
+            # Make sure it's a geometric (meters-based) CRS
+            ROI = ensure_projected_CRS(ROI)
+            # Read the locations of all the points
+            # TODO do these need to be swapped
```

Comment:
I suspect that this TODO is outdated and we don't need to swap the order of the lat-lon values. Can you confirm that all the conventions match up and then remove this comment?